app/services/tickets/v1/index.rb

```diff
@@ -52,19 +52,19 @@ def call
         tickets = Ticket.where(requester_id: @current_user.id)
       elsif @current_user.is_employee?
         tickets = Ticket.where(requester_id: @current_user.id)
-      elsif @current_user.is_resolver?
-        if !@assigned_to_me && !@created_by_me && where_hash.length == 1
-          tickets = Ticket.where(resolver_id: @current_user.id).or(Ticket.where(requester_id: @current_user.id))
-        elsif !@assigned_to_me && !@created_by_me && where_hash.length > 1
-          tickets = Ticket.where(where_hash).and(Ticket.where(resolver_id: @current_user.id))
-        elsif @assigned_to_me && @created_by_me && where_hash.length > 1
-          tickets = Ticket.where(where_hash).and((Ticket.where(requester_id: @current_user.id)).or\
-                                                 (Ticket.where(resolver_id: @current_user.id)))
-        elsif @assigned_to_me && where_hash.length > 1
-          tickets = Ticket.where(where_hash).and(Ticket.where(resolver_id: @current_user.id))
-        elsif @created_by_me && where_hash.length > 1
-          tickets = Ticket.where(where_hash).and(Ticket.where(requester_id: @current_user.id))
-        end
+      # elsif @current_user.is_resolver?
```

Comment:
Are you sure about commenting this, can other role users perform tasks of resolver?

Reply:
There is no any user as resolver in our system it will be either department_head or admin and employee so there won't be any issue for this